tdrs-backend/tdpservice/security/models.py

```diff
@@ -0,0 +1,114 @@
+"""Models for the tdpservice.security app."""
+from hashlib import sha256
+import logging
+
+from django.contrib.admin.models import ContentType, LogEntry, ADDITION
+from django.core.files.base import File
+from django.db import models
+
+from tdpservice.data_files.models import DataFile
+from tdpservice.users.models import User
+
+logger = logging.getLogger(__name__)
+
+
+class ClamAVFileScanManager(models.Manager):
+    """Extends object manager functionality with common operations."""
+
+    def record_scan(
```

Comment:
This is a helper function to easily create a `ClamAVFileScan` instance at the same time as a linked `LogEntry`.